### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by silverxp**
Validation.php needs review in the future, but can't be translated for now.

### CODE REVIEW

**Comment by silverxp** [comment]
Closing.

**Comment by silverxp** [comment]
Closing.

File: app/config/langs.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
     'pl'      => 'Polski',
     'es'      => 'Español',
     'fr'      => 'Français',
+	'nl'      => 'Nederlands',
```
Comment (jbrooksuk) [inline app/config/langs.php]: Too much indentation here.

**Comment by silverxp** [comment]
Fixed.

**Comment by jbrooksuk** [comment]
You're awesome @silverxp, thank you!

**Comment by silverxp** [comment]
Glad I could help @jbrooksuk :)

**Pull request merged**